### ISSUE TITLE
pick_by has different behaviour with lodash.pickBy

### ISSUE THREAD

**Issue by cnwangjie** [dgilland/pydash#144]
```
In [1]: import pydash

In [2]: pydash.pick_by({'a': 1, 'b': 0})
Out[2]: {}

In [3]: pydash.pick_by({'a': 1, 'b': 0}, lambda i: i)
Out[3]: {'a': 1}

```
Expected behaviour: `pydash.pick_by({'a': 1, 'b': 0})` will return `{'a': 1}`

Actual behaviour: return `{}`

Following is the behaviour in Lodash
```
> _.pickBy({a: 1, b: 0})
< {a: 1}
```

**Comment by slallum** [comment]
duplicate of #129, closed by #139 :)

**Comment by cnwangjie** [comment]
Thanks
